srf_generation/velocity_model_generation/q_model_generation/EP_to_bin.py

```diff
@@ -0,0 +1,51 @@
+"""Script to generate a Qp/Qs binary model file from an Eberhart-Phillips style ascii model"""
+import argparse
+from os.path import abspath
+import pandas as pd
+
+from qcore.vm_file import VelocityModelFile
+
+
+def load_args():
+    parser = argparse.ArgumentParser(allow_abbrev=False)
+    parser.add_argument("infile", type=abspath)
+    parser.add_argument("outfile", type=abspath)
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = load_args()
+
+    # load model data
+    data = pd.read_csv(args.infile, sep="\s+", skiprows=1, header=0, index_col=None)
+    data.columns = ["vals", "SF", "x", "y", "z", "lat", "lon"]
+    data.sort_values(["x", "y", "z"], axis=0, inplace=True)
+    data.reset_index(inplace=True, drop=True)
+    xs = data["x"].unique()
+    ys = data["y"].unique()
+    zs = data["z"].unique()
+    xs_count = len(xs)
```

Comment:
What about?
```
xs_count, ys_count, zs_count = len(xs), len(ys), len(zs)
```
Especially for such simple repeated variable definitions?

Reply:
Huh, I don't usually think of putting that sort of thing on one line
I normally only use that notation for tuple/list packing/unpacking
I could change it, but you'd have to approve it again